portfolio/src/main/webapp/script.js

```diff
@@ -70,19 +70,16 @@ function addRandomFact() {
  * Fills in the 'I am a' sentences with different phrases
  */
 function writeSnippets() {
-    if (i == 0) document.getElementById('snippetsTarget').innerHTML = EMOJIS[j] + ' I am ';
+    if (charIdx == 0) document.getElementById('snippetsTarget').innerHTML = EMOJIS[phraseIdx] + ' I am ';
 
-	// Use typing effect to print PHRASE[j]
-	if (i < PHRASES[j].length) {
-        document.getElementById('snippetsTarget').innerHTML += PHRASES[j].charAt(i);
-		i++;
+	// Use typing effect to print PHRASE[phraseIdx]
+	if (charIdx < PHRASES[phraseIdx].length) {
+        document.getElementById('snippetsTarget').innerHTML += PHRASES[phraseIdx].charAt(charIdx);
+		charIdx++;
 		setTimeout(writeSnippets, RATE);
-	} 
-
-	// Go to next phrase after entire phrase has been printer
-	else {
-		i = 0;
-        (j + 1) < PHRASES.length ? j++ : j = 0;
+	} else { // Go to next phrase after entire phrase has been printer
```

Comment:
the body of this if statement doesn't line up
`charIdx++;` and the `setTimeout` call should be de-indented

also, maybe it should be `charIndex`?

also, don't use innerHTML here either

Reply:
in terms of indentation... in the cloud shell text editor its all lined up (could there be some issue that is changing the indentation between cloud shell and github)

Reply:
I've seen issues like that before when copying code to a new editor. usually it's related to tabs vs spaces. so my advice is to un-indent everything so there's no space at all, then re-indent with tab (or just something consistent) so things line up. that usually works for me.